.travis.yml

```diff
@@ -68,19 +70,13 @@ jobs:
     - <<: *bincompat_check
       scala: *scala_version_212
 
-    - &publish_snapshot
-      stage: publish snapshot
+    - stage: publish snapshot
       script: |
         if [[ $(cat version.sbt) =~ "-SNAPSHOT" ]]; then
-          sbt ++$TRAVIS_SCALA_VERSION! publish gitSnapshots publish
+          sbt +publish gitSnapshots +publish
```

Comment:
It makes it possible for each module to define which scala version to release on (based on their own `crossScalaVersions`), also simplifies the build here.